### DISCUSSION TITLE
废话杂谈

### DISCUSSION THREAD

**Discussion by Minemetero** [lingbopro/easy-github-hosts#13] [General]
一群活跃的开发者，避免不了废话

**Comment by Minemetero** [comment]
![Screenshot 2024-05-15 8 47 04 AM](https://github.com/lingbopro/easy-github-hosts/assets/156386015/1b063e13-4635-458f-a701-09e44670369c)
我和merge的时候把两条给合并了看着好难受aaaaaaaaaaaaaaaa

**Reply by lingbopro** [reply]
悲

**Comment by Minemetero** [comment]
哥们,动起来啊...
10年没开张了啊

**Reply by lingbopro** [reply]
啊

**Reply by lingbopro** [reply]
我出了个新repo

**Reply by Minemetero** [reply]
oh？

**Comment by Minemetero** [comment]
<img width="233" alt="屏幕截图 2024-05-26 203736" src="https://github.com/lingbopro/easy-github-hosts/assets/156386015/59eb7cb5-c455-4aba-aca9-2ffaad1d98dc">
这™是什么鬼东西啊

**Reply by lingbopro** [reply]
> <img alt="屏幕截图 2024-05-26 203736" width="233" src="https://private-user-images.githubusercontent.com/156386015/333875941-59eb7cb5-c455-4aba-aca9-2ffaad1d98dc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY3NzY4MDMsIm5iZiI6MTcxNjc3NjUwMywicGF0aCI6Ii8xNTYzODYwMTUvMzMzODc1OTQxLTU5ZWI3Y2I1LWM0NTUtNGFiYS1hY2E5LTJmZmFhZDFkOThkYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUyN1QwMjIxNDNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lNjc4ZWNiYzJiZDFmYmViY2RkZmIyNjBmMGI0ZDJhMjY5NTExZWEwYmM3ZDQ5ODBjNThlZTlkZDBjOTY3Y2Q1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.dOzUFYfCZrtR1wPiCkde0bF9lP7J-J9z22FvxCNanwM"> 这™是什么鬼东西啊

GitHub 特有的奇妙用户（

**Comment by lingbopro** [comment]
> <img width="233" alt="屏幕截图 2024-05-26 203736" src="https://github.com/lingbopro/easy-github-hosts/assets/156386015/59eb7cb5-c455-4aba-aca9-2ffaad1d98dc">
> 这™是什么鬼东西啊

我大概知道这是什么情况了  

![Screenshot_20240527_175411.jpg](https://github.com/lingbopro/easy-github-hosts/assets/148617407/eea4cd3e-c639-4dd8-895b-00580e6fac29)

**Reply by Minemetero** [reply]
<img width="157" alt="屏幕截图 2024-05-27 200417" src="https://github.com/lingbopro/easy-github-hosts/assets/156386015/285f4fa7-cd90-4de7-bd6b-c804c583f5bd">
ahhhhhhhh

**Reply by lingbopro** [reply]
> <img width="157" alt="屏幕截图 2024-05-27 200417" src="https://github.com/lingbopro/easy-github-hosts/assets/156386015/285f4fa7-cd90-4de7-bd6b-c804c583f5bd">
> ahhhhhhhh
> 

啊

**Comment by Minemetero** [comment]
邮箱少打了个o

**Reply by lingbopro** [reply]
emmm

**Comment by Minemetero** [comment]
1.1.0的时候其实加的内容不足以升一个版本号所以我集体给他们降了一个版本号。（主要是闲的。。。。）你这一次的版本号用1.3.0

**Reply by lingbopro** [reply]
这不好吧…   
已经发了的版本就别改了

**Comment by Minemetero** [comment]
5555555, 我搞了个新repo然后Score按键不工作了。5555555555555555555555555555，前面还好好的，可不可以帮忙修修 ：）

**Reply by lingbopro** [reply]
过两天做功能的时候顺便看看罢:)

**Reply by Minemetero** [reply]
打开consle看到了问题正在修复。。。

**Reply by Minemetero** [reply]
修好啦

**Comment by Minemetero** [comment]
![图像_2024-06-05_114837160](https://github.com/lingbopro/easy-github-hosts/assets/156386015/db57378b-0df2-4460-84b7-23a55b88bc52)
hummmmm我像来搞破坏的

**Reply by lingbopro** [reply]
# 🤔

**Comment by Minemetero** [comment]
什么人可以在在发完PR之后快速下线的。。

**Reply by lingbopro** [reply]
> 什么人可以在在发完PR之后快速下线的。。

me :)

**Comment by Minemetero** [comment]
![image_2024-06-09_112712858](https://github.com/lingbopro/easy-github-hosts/assets/156386015/3e5f1797-8dc1-4de2-8d3a-519497f224d9)
这很有意思。。。。

**Reply by lingbopro** [reply]
？这是啥

**Reply by Minemetero** [reply]
那个你去insights里面有个traffic里面有我们这个repo的访客

**Reply by lingbopro** [reply]
> 那个你去insights里面有个traffic里面有我们这个repo的访客

哦  
所以这个`dvel.me`是啥

**Reply by Minemetero** [reply]
我也不知道可能是某个爬虫爬过。。。。

**Reply by lingbopro** [reply]
访问一下这个域名不就知道了（

**Reply by lingbopro** [reply]
<img width="864" alt="image" src="https://github.com/lingbopro/easy-github-hosts/assets/148617407/f2db569e-d380-420a-a976-ad4cee5b247f">

起码能ping通  
直接访问好像有点草率啊（（（

**Reply by Minemetero** [reply]
不是可以访问吗？还挺好看的

**Reply by Minemetero** [reply]
是一个人的blog

**Reply by lingbopro** [reply]
> 不是可以访问吗？还挺好看的

主要是我不知道这个网站是啥

**Reply by lingbopro** [reply]
> 是一个人的blog

问题就在这了  
既然是blog  
为什么会有对这个repo的访问，referer指向这个blog

**Reply by Minemetero** [reply]
就很奇怪，这也是为啥我说 ‘这很有意思。。。。’

**Comment by lingbopro** [comment]
在OJ写题的我：  

<img width="1067" alt="114514" src="https://github.com/lingbopro/easy-github-hosts/assets/148617407/7a77fb25-e270-4db0-a0c0-c9b96ef2bdc6">

过 于 惨 烈 (bushi

**Reply by Minemetero** [reply]
hehe

**Reply by Minemetero** [reply]
高达惊人的 8%

**Comment by lingbopro** [comment]
![image](https://github.com/lingbopro/easy-github-hosts/assets/148617407/402ff2ff-b46d-4b90-805a-179272b72b4a)

关于我们这个repo竟然有20个人clone，11个人访问

**Reply by Minemetero** [reply]
emmmm,他们都是聋哑人。。。。

**Reply by Minemetero** [reply]
clone是只需要clone就加1所以这大概率是我们两一直克隆所以就20个了，毕竟也没有其他人在开发。。。

**Reply by lingbopro** [reply]
问题在于我只在我的电脑和学校的电脑上clone过啊

**Comment by Minemetero** [comment]
相比于最开始的代码改进很大嘛。。。

**Comment by Minemetero** [comment]
![image_2024-06-17_084700950](https://github.com/lingbopro/easy-github-hosts/assets/156386015/42948dc1-a9b2-4d90-bbe0-cbe58710bfc3)
5 star 这群人真奇怪

**Reply by Minemetero** [reply]
![image_2024-06-17_091607875](https://github.com/lingbopro/easy-github-hosts/assets/156386015/bcf6af67-e4dc-48f3-b323-e0f4035f8d2c)
他给了你star这很不公平 （bushi

**Reply by lingbopro** [reply]
我刚起床啊  
话说这是哪个repo啊  
就easy-github-hosts吗

**Reply by lingbopro** [reply]
> ![image_2024-06-17_091607875](https://github.com/lingbopro/easy-github-hosts/assets/156386015/bcf6af67-e4dc-48f3-b323-e0f4035f8d2c)
> 他给了你star这很不公平 （bushi

这人的repo咋全是iptv啊（

**Reply by Minemetero** [reply]
热爱刷剧的人

**Reply by Minemetero** [reply]
我要开新repo啦

**Reply by lingbopro** [reply]
> 我要开新repo啦

o  
啥repo啊

**Reply by lingbopro** [reply]
> 我要开新repo啦

看见了  
不过readme没更新啊（恼

**Reply by Minemetero** [reply]
马上更新

**Comment by Minemetero** [comment]


**Reply by lingbopro** [reply]
这...唬人？

**Comment by Minemetero** [comment]
1.4.2没改啥，这些代码早就写好了一直没推上来

**Comment by Minemetero** [comment]
这个repo差不多就这样了。。。。进入维护阶段了

**Comment by Minemetero** [comment]
![image_2024-07-11_222457801](https://github.com/lingbopro/easy-github-hosts/assets/156386015/aaa140e6-1b0a-4282-9b69-e089a209e592)
一看就是顺藤摸瓜从win12找过来的

**Reply by lingbopro** [reply]
啊～

**Reply by zsr-lukezhang** [reply]
啊？

**Reply by lingbopro** [reply]
我在win12还回复过他

**Reply by zsr-lukezhang** [reply]
对啊，没错

**Reply by kjmjh** [reply]
我也闻着味来了~

**Comment by zsr-lukezhang** [comment]
我什么都不会~

**Reply by Minemetero** [reply]
额大佬谦虚了

**Comment by zsr-lukezhang** [comment]
全是Forked的（悲）

**Comment by zsr-lukezhang** [comment]
不要猜错了😅

**Reply by Minemetero** [reply]
额啥意思

**Comment by zsr-lukezhang** [comment]
这些似乎好像都是复刻的

**Reply by zsr-lukezhang** [reply]
只是没留痕

**Reply by Minemetero** [reply]
：（

**Reply by Minemetero** [reply]
没事从现在开始学

**Reply by Minemetero** [reply]
加入我们的开发团队 :) 我真不把自己当外人（

**Reply by zsr-lukezhang** [reply]
啊这... （你看）

**Reply by Minemetero** [reply]
管他呢反正我是这里的协作者，有权限

**Reply by lingbopro** [reply]
> ：（

《  中  文  符  号  》

**Comment by lingbopro** [comment]
![Screenshot_20240712_153504.jpg](https://github.com/user-attachments/assets/bbed3eec-8c88-4b97-9e7b-691e71070418)

我的follower突然就变多了

**Reply by Minemetero** [reply]
羡慕 嫉妒 恨

**Reply by lingbopro** [reply]
你终于回来了啊

**Reply by Minemetero** [reply]
不懂就问，为啥CSS的文件是globle开头

**Reply by Minemetero** [reply]
而不是components

**Reply by lingbopro** [reply]
> 不懂就问，为啥CSS的文件是globle开头

只是一个名字罢了

**Reply by lingbopro** [reply]
> 而不是components

这个是全局变量的定义
components那个是组件的

**Comment by lingbopro** [comment]
现在我也有非常棒的README了:)

**Reply by Minemetero** [reply]
这就去抄袭

**Reply by Minemetero** [reply]
不对，好像没啥好抄的。。。。

**Reply by Minemetero** [reply]
你全是在抄我啊

**Reply by lingbopro** [reply]
也不能这么说

**Reply by lingbopro** [reply]
Metrics我改了不少

**Reply by Minemetero** [reply]
（

**Reply by lingbopro** [reply]
而且我的生成的文件都在output分支，不会影响主分支:)

**Reply by Minemetero** [reply]
你还抄我metrics

**Reply by lingbopro** [reply]
另外
我发现你的README的Metrics因为没给token一直失败啊

**Reply by lingbopro** [reply]
> 你还抄我metrics

本来就是来源项目
谁都可以用嘛（

**Reply by Minemetero** [reply]
确实

**Comment by lingbopro** [comment]
我现在正在做新项目

**Reply by Minemetero** [reply]
啥项目。。。你创意从来不会断。。。我一直想不出我要做啥

**Reply by Minemetero** [reply]
我感觉我是个打杂的，到处缝缝补补。。。

**Reply by lingbopro** [reply]
一个Node.js做的东西

**Comment by lingbopro** [comment]
我抽空玩了一下Minetest `:)`

**Reply by Minemetero** [reply]
我要做Fabric模组你有啥推荐的工具吗

**Reply by lingbopro** [reply]
> 我要做Fabric模组你有啥推荐的工具吗

IDEA?

**Reply by lingbopro** [reply]
我用的IDEA反正

**Reply by Minemetero** [reply]
我知道，我是说Ui设计啥的

**Reply by Minemetero** [reply]
我出了个repo还只是个概念 :)

**Reply by lingbopro** [reply]
> 我知道，我是说Ui设计啥的

我不到啊

**Reply by lingbopro** [reply]
> 我出了个repo还只是个概念 :)

哦？

**Reply by lingbopro** [reply]
不管是啥，反正先Star一下（

**Reply by Minemetero** [reply]
好吧。。。。真正出估计要到明年了

**Reply by Minemetero** [reply]
我还要学学Java

**Reply by Minemetero** [reply]
这将会是一个作弊mod....你想好了再star ;)

**Reply by lingbopro** [reply]
> 这将会是一个作弊mod....你想好了再star ;)

是个mod就行
更何况还是来源的:P

**Reply by lingbopro** [reply]
![Blue :P](https://github.com/user-attachments/assets/22517dc1-39b9-4f29-8cf2-f93fa9ff1249)

**Comment by lingbopro** [comment]
我不想做我的新项目了我要做mod

![>:P ←:D ←:D ←:D ←:D](https://github.com/user-attachments/assets/b22f9c62-9381-4c46-8b36-6c4d6a6ca8a9)

`>:P ←:D ←:D ←:D ←:D`

**Reply by Minemetero** [reply]
嘲笑

**Reply by Minemetero** [reply]
Java比Python难学多了

**Reply by Minemetero** [reply]
做一个项目最好的方式，是了解项目。我在MC里无法自拔。。。

**Reply by lingbopro** [reply]
> 做一个项目最好的方式，是了解项目。我在MC里无法自拔。。。

6

**Reply by lingbopro** [reply]
我目前有点进展（指下载了示例模板然后改了几个字符串

**Reply by Minemetero** [reply]
你开个Private的repo共享给我，我们一起学，一起做

**Reply by lingbopro** [reply]
但是我写的Forge

**Reply by Minemetero** [reply]
:(

**Reply by Minemetero** [reply]
Forge 用的人不多了了，被淘汰了

**Reply by Minemetero** [reply]
NeoForge 是新版本对 Forge的替代品

**Reply by Minemetero** [reply]
NeoForge 的创始人曾经维护过Forge，他觉得Forge过于沉重开发出了NeoForge

**Reply by lingbopro** [reply]
虽然但是，不会NeoForge

**Reply by lingbopro** [reply]
而且NeoForge尚不成熟

**Reply by Minemetero** [reply]
已经很多mod在支持NeoForge了,都在支持NeoForge的路上

**Reply by lingbopro** [reply]
我的mod里一个方块拿在手上是正常的，放下来纹理就出问题了…
绝对是forge的问题，绝对不是因为我为了省事直接复制原版的东西然后纹理图层指定错了（

**Reply by Minemetero** [reply]
我觉得选择路线是 Fabric>NeoForge>Forge

**Reply by lingbopro** [reply]
但是我选择Forge（

**Reply by lingbopro** [reply]
> 我的mod里一个方块拿在手上是正常的，放下来纹理就出问题了…
> 绝对是forge的问题，绝对不是因为我为了省事直接复制原版的东西然后纹理图层指定错了（

我说错了，是纹理图层路径忘加`block/`了

**Comment by lingbopro** [comment]
我的Mod发布力！

[lingbopro/lingbos-sussy-mod](https://GitHub.com/lingbopro/lingbos-sussy-mod)

**Reply by Minemetero** [reply]
............. 无语至极的mod

**Reply by Minemetero** [reply]
如果，我说如果，我开一个Minecraft服务器你会愿意做开发吗

**Reply by lingbopro** [reply]
> 如果，我说如果，我开一个Minecraft服务器你会愿意做开发吗

当然:)

**Reply by lingbopro** [reply]
我曾经的梦想就是：开一个Minecraft服务器，如果我很行的话就开个工作室（

**Reply by zsr-lukezhang** [reply]
Edge居然给我卡bug了
<img width="1080" alt="image" src="https://github.com/user-attachments/assets/ec86830c-65d3-4a27-9bd4-56921d566351">
还有，这么快，就有人开Issue了？！

**Reply by zsr-lukezhang** [reply]
指的是地址栏 about:blank

**Reply by Minemetero** [reply]
> 我曾经的梦想就是：开一个Minecraft服务器，如果我很行的话就开个工作室（

：）

**Reply by Minemetero** [reply]
> Edge居然给我卡bug了 <img alt="image" width="1080" src="https://private-user-images.githubusercontent.com/145239958/356630937-ec86830c-65d3-4a27-9bd4-56921d566351.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMyNTE1MTUsIm5iZiI6MTcyMzI1MTIxNSwicGF0aCI6Ii8xNDUyMzk5NTgvMzU2NjMwOTM3LWVjODY4MzBjLTY1ZDMtNGEyNy05YmQ0LTU2OTIxZDU2NjM1MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgxMFQwMDUzMzVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02MDcxY2VhYmZlNjIzNTI3N2Y5MmU4ZTk3YzFlOWMxOWQ4OGE3YzBmNDg0ZjcxZDI2MGVjYWEyMzE0ZTBiMmE5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.2cysj__u1t_IJ04OSTBcIxONXL6ny1U47hytT2vCajs"> 还有，这么快，就有人开Issue了？！

哪里是issue了

**Reply by lingbopro** [reply]
> > Edge居然给我卡bug了 <img alt="image" width="1080" src="https://private-user-images.githubusercontent.com/145239958/356630937-ec86830c-65d3-4a27-9bd4-56921d566351.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMyNTE1MTUsIm5iZiI6MTcyMzI1MTIxNSwicGF0aCI6Ii8xNDUyMzk5NTgvMzU2NjMwOTM3LWVjODY4MzBjLTY1ZDMtNGEyNy05YmQ0LTU2OTIxZDU2NjM1MS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgxMFQwMDUzMzVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02MDcxY2VhYmZlNjIzNTI3N2Y5MmU4ZTk3YzFlOWMxOWQ4OGE3YzBmNDg0ZjcxZDI2MGVjYWEyMzE0ZTBiMmE5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.2cysj__u1t_IJ04OSTBcIxONXL6ny1U47hytT2vCajs"> 还有，这么快，就有人开Issue了？！
> 
> 哪里是issue了

我Mod的repo？

**Comment by Minemetero** [comment]
![image_2024-08-28_090650696](https://github.com/user-attachments/assets/5ac1d777-4100-4449-b57d-2d68fe499219)
ntm Fork这个干啥 我在班上打开来看一眼尴尬死了

**Reply by Minemetero** [reply]
cnmd

**Reply by lingbopro** [reply]
因为我有很多图:D

**Reply by kjmjh** [reply]
哈哈😄

**Comment by zsr-lukezhang** [comment]
@lingbopro ？？？
![image](https://github.com/user-attachments/assets/98d74d99-72eb-4447-b767-ebf7a99d1023)

**Reply by lingbopro** [reply]
没问题啊

**Reply by zsr-lukezhang** [reply]
> 没问题啊

你的README里好像没有提B站主页
只提了有B站账号

**Reply by lingbopro** [reply]
我提了

![image](https://github.com/user-attachments/assets/b666a673-fa24-4bca-a9c9-2cdeffffeaa6)

我README里的这些Badge都是链接

**Reply by Minemetero** [reply]
oh?

**Comment by lingbopro** [comment]
我给我的README的工作流里放了点私货（（（（（

**Reply by Minemetero** [reply]
oh? 这就去看看

**Comment by Minemetero** [comment]
died chat

**Reply by lingbopro** [reply]
https://github.com/Runixe786/MD3-Windows

看到一个好东西

**Reply by Minemetero** [reply]
这。。这不是ChromeOS吗

**Reply by zsr-lukezhang** [reply]
> 这。。这不是ChromeOS吗

看了一眼UI还真是

**Reply by zsr-lukezhang** [reply]
> https://github.com/Runixe786/MD3-Windows
> 
> 看到一个好东西

这玩意我刚才试了一下（对没错这么远古的东西我刚尝试
清晰度感人
我关掉高DPI试一下......
哦，果然更清晰了（
![image](https://github.com/user-attachments/assets/a5402622-fbb1-43c4-8674-e2d8ce8fa332)

**Comment by lingbopro** [comment]
试图在手机的Termux上安装ollama并跑模型ing...

**Reply by Minemetero** [reply]
你先帮我把模型给画了

**Reply by Minemetero** [reply]
那个模组已经搁置很久了

**Reply by lingbopro** [reply]
![image](https://github.com/user-attachments/assets/e8cc961b-18ed-413e-9be2-a9cd90f7efaa)

这样彳亍嘛（还没做完

**Reply by lingbopro** [reply]
![image](https://github.com/user-attachments/assets/dfb1c7cf-a531-4284-aaf1-9c2ef1b33677)

做完了，行嘛

**Reply by Minemetero** [reply]
可以藏点啥小彩蛋

**Reply by Minemetero** [reply]
或者搞特殊点，外观搞一搞

**Reply by Minemetero** [reply]
无论如何，这张已经够用了

**Reply by lingbopro** [reply]
彳亍，等我中午回来导出

**Reply by lingbopro** [reply]
![Minemetero/Hello-IPs](https://github.com/user-attachments/assets/851e35b4-053c-4bef-a5f2-6c9d45058319)

![Minemetero/BlackJack](https://github.com/user-attachments/assets/8a363c55-04bd-42a8-b3c4-0cdbb78fb285)

oh?

**Reply by Minemetero** [reply]
开发中

**Comment by lingbopro** [comment]
![image](https://github.com/user-attachments/assets/cbcb7f4e-9933-4eed-b2a6-7b2eaed89c23)

WSL 的龟速下载

**Reply by zsr-lukezhang** [reply]
你去香港不就行了（

**Reply by lingbopro** [reply]
> 你去香港不就行了（

```js
我去香港的花费 > 我等下载完的花费
```

**Reply by zsr-lukezhang** [reply]
> > 你去香港不就行了（
> 
> ```js
> 我去香港的花费 > 我等下载完的花费
> ```

```
```js
```
？？？为啥是JS

**Reply by zsr-lukezhang** [reply]
> 你去香港不就行了（

或者，请看emoji

**Comment by Minemetero** [comment]
![image_2024-12-17_214011575](https://github.com/user-attachments/assets/64146298-aa66-4124-bf78-f32136da1269)
大家对外挂的需求似乎有点大

**Reply by Minemetero** [reply]
有个meteor client的插件achive了,我维护了一下修了点bug,然后来了一堆人访问

**Reply by Minemetero** [reply]
这是我所有repo中见过最大的view了 :sob:

**Reply by lingbopro** [reply]
> 有个meteor client的插件achive了,我维护了一下修了点bug,然后来了一堆人访问

那我也去看一下（

**Comment by Minemetero** [comment]
我无语了。。。为什么这个repo还有人fork啊，15个星。。开什么玩笑啊

**Comment by zsr-lukezhang** [comment]
@lingbopro 给个写的权限（

**Reply by zsr-lukezhang** [reply]
我在改这玩意，懒得弄 PR 了

**Reply by zsr-lukezhang** [reply]
@lingbopro @lingbopro @lingbopro @lingbopro @lingbopro

**Reply by zsr-lukezhang** [reply]
eeee

**Reply by lingbopro** [reply]
emmmmmm,才看见

虽然已经开PR了但是如果你想要的话可以给你 :)

**Reply by Minemetero** [reply]
我找个良辰吉日把代码重构了

**Reply by lingbopro** [reply]
> 我找个良辰吉日把代码重构了

重构第一步：弃用CJS，转用ESM（

**Comment by Minemetero** [comment]
![image_2025-01-23_085154584](https://github.com/user-attachments/assets/d65cbd6f-7323-462b-9d36-fb92e4c7cabb)
我提议把他权限给下了

**Reply by Minemetero** [reply]
不过确实我们规范太少了点

**Reply by lingbopro** [reply]
emmm?

**Comment by Minemetero** [comment]
![image_2025-01-23_164654480](https://github.com/user-attachments/assets/b49f30b3-6457-4418-af7c-0bb3ea053094)
这个人是谁？

**Reply by lingbopro** [reply]
idk

**Comment by Minemetero** [comment]
为什么这个人还有一个commit我怎么不知道？

**Comment by Minemetero** [comment]
早期lingbopro代码
[History.zip](https://github.com/user-attachments/files/18529226/History.zip)

**Comment by Minemetero** [comment]
来人发个Release啊

**Reply by Minemetero** [reply]
怎么都变死人了。。。。。。

**Comment by Minemetero** [comment]
好家伙 17 颗星

**Reply by Minemetero** [reply]
我也想要那个星星的徽章 :/

**Reply by lingbopro** [reply]
我都没意识到我竟然有这个徽章（）

**Comment by Minemetero** [comment]
这个仓库简直是尴尬，那个时候啥也不会。
话说我们这1.5.0都做出来那么久了，没人发release

**Reply by lingbopro** [reply]
emmmmmmmm

**Comment by kjmjh** [comment]
我来了(闲得慌

**Reply by Minemetero** [reply]
你看看看我的Hello-IPs吧

**Comment by lingbopro** [comment]
👍

![image](https://github.com/user-attachments/assets/448a8f99-0f43-4825-9016-6fa0f816dcc5)

**Reply by Minemetero** [reply]
什么

**Reply by Minemetero** [reply]
咋得到的？？？？？？？？

**Reply by lingbopro** [reply]
> 咋得到的？？？？？？？？

去Epic绑定一下GitHub就会给你发邀请，加入组织就算绑定成功

**Reply by Minemetero** [reply]
666666吓死我了，以为你成Epic的开发了 hahahaha

**Comment by Minemetero** [comment]
@lingbopro sjt的号。。好像没了。。

**Reply by lingbopro** [reply]
啊？

**Reply by lingbopro** [reply]
还在啊

![Screenshot_20250612_191326_com.github.android.jpg](https://github.com/user-attachments/assets/bf27e386-8327-4b5c-b4df-5a24506d1af8)

**Reply by kjmjh** [reply]
改了个名而已。。。

**Reply by Minemetero** [reply]
oh 你的README得改了

**Reply by kjmjh** [reply]
@lingbopro  ？

**Reply by lingbopro** [reply]
不过说实话我对sjt改用户名不太赞同，毕竟GitHub文档都说了改用户名可能会造成一些不可预见的影响

**Reply by lingbopro** [reply]
> > oh 你的README得改了
> 
 see commit 820b7d0421adf8828f406b034ffd6a5faa41a176

**Reply by lingbopro** [reply]
> oh 你的README得改了

https://github.com/lingbopro/lingbopro/commit/820b7d0421adf8828f406b034ffd6a5faa41a176
